src/main/java/com/configcat/ClientReadyState.java

```diff
@@ -0,0 +1,23 @@
+package com.configcat;
+
+/**
+ * Describes the Client state.
+ */
+public enum ClientReadyState {
```

Comment:
For consistency, I suggest the name `ClientCacheState`, this is used in .NET and JS SDKs (unfortunately, not in Swift SDK).

However, according to the plans, snapshot API will be implemented in Java eventually and snapshots will expose this information too. In that case the suggested name makes more sense. See https://github.com/configcat/.net-sdk/blob/v9.2.0/src/ConfigCatClient/ConfigCatClientSnapshot.cs#L41